javascript-timers/interval.js

```diff
@@ -0,0 +1,18 @@
+const header = document.querySelector('.countdown-display');
+let interval = 4;
+
+const id = setInterval(() => {
+  if (interval === 4) {
+    interval--;
+    header.textContent = 3;
+  } else if (interval === 3) {
+    interval--;
+    header.textContent = 2;
+  } else if (interval === 2) {
+    interval--;
+    header.textContent = 1;
```

Comment:
It seems like almost the same operation is being used for each of these cases. How would you combine them into one case?